iosxr_test.py

```diff
@@ -276,6 +293,9 @@ def main(vbox=None, verbosity=logging.INFO):
         raise AbortScriptException('%s does not exist' % input_box)
 
     logger.setLevel(level=verbose)
+    # Since we are importing methods from the main iso2vbox module,
+	# ensure its logging is also set according to our preference.
```

Comment:
Yup, @#$@#$ tab character crept in here because the server I was testing these changes on didn't have sane `vim` configuration. Will fix.